<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-EU5RSED" ignored="0" name="Mult Cal Test" tests="8" failures="1" timestamp="2019-08-20T20:42:18 GMT+04:00" time="0.008" errors="0">
  <testcase name="doubleOneMultTwoTest" time="0.004" classname="com.epam.cdp.tests.testng.MultCalTest">
    <failure type="java.lang.AssertionError" message="Invalid result of mult operation! did not expect to find [4.5] but found [4.0]">
      <![CDATA[java.lang.AssertionError: Invalid result of mult operation! did not expect to find [4.5] but found [4.0]
at com.epam.cdp.tests.testng.MultCalTest.doubleOneMultTwoTest(MultCalTest.java:26)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.epam.cdp.tests.testng.Runner.main(Runner.java:24)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doubleOneMultTwoTest -->
  <testcase name="doubleOneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="doubleOneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="doubleOneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="oneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="oneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="oneMultTwoTest" time="0.0" classname="com.epam.cdp.tests.testng.MultCalTest"/>
  <testcase name="oneMultTwoTest" time="0.004" classname="com.epam.cdp.tests.testng.MultCalTest"/>
</testsuite> <!-- Mult Cal Test -->
